colonel/wrapper/wrapper.py

```diff
@@ -78,8 +79,6 @@ def __init__(self):
 
     def discover_executable_route(self):
         discoverer = CompoundExecutableDiscoverer(
-            PathEnvironmentVarExecutableDiscoverer(),
-            LibraryEnvironmentVarExecutableDiscoverer(),
```

Comment:
should we delete these classes??

Reply:
Yes. It makes me 😢, but it makes the coverage report look like shit.